src/templates/base-generated-config.yaml

```diff
@@ -86,6 +91,7 @@ jobs:
             else
               curl -v -F buildingBlock=${CIRCLE_PROJECT_REPONAME} \
                 -F report=@${TEST_RESULT_REPORT_PATH}             \
+                -F version=<< parameters.bb-version >>            \  # Adding tag as version
```

Comment:
Is it handled on backend of testing webapp?

Reply:
Yes